webidl/Generate.hx

```diff
@@ -412,26 +412,32 @@ template<typename T> pref<T> *_alloc_const( const T *value ) {
 		var outFiles = [];
 		sources.push(lib+".cpp");
 		for( cfile in sources ) {
-			var out = cfile.substr(0, -4) + ".bc";
-			var args = params.concat(["-c", cfile, "-o", out]);
+			var sourcePath = if(sys.FileSystem.exists(opts.out+ "/" + cfile)){

```

Comment:
I might be missing something, but why are we looking in the output directory as a possible location for a source file?

Reply:
Because it is, the only place where the lib_project.cpp is going to ever exist is in the output directory; it's generated.

Reply:
In order to preserve the sources separate from the output dir concept, which I think is good, we could create like a "bundled" directory maybe and add that to the `webidl.json` options?

Reply:
That makes sense, in that case I would just update the if statement to be single lined:

```haxe
var sourcePath = sys.FileSystem.exists(opts.out + "/" + cfile) ? opts.out + "/" + cfile : cfile;
````